### PR TITLE
import process fixes

### PR DESCRIPTION
**Pull request by tkimnguyen**
- [x] transition item only if it needs it
- [x] loop over all workflow chains (usually there is only one)
- [x] set the workflow history (do not add extraneous entries caused by the import process)

### CODE REVIEW

File: castle/cms/_scripts/importjson.py [hunk 1]
```diff
@@ -346,38 +346,38 @@ def import_object(filepath):
...
-            if len(chain):
-                workflow_id = chain[0]
-                for h in review_history:
-                    wtool.setStatusOf(workflow_id, obj, h)
-                if len(chain) != 1:
-                    logger.warn('There should be 1 workflow for %s but'
-                                        'there are %i' % (path, len(chain)))
+            # loop over all workflow chains (usually only 1)
+            for workflow_id in wtool.getChainFor(obj):
+                obj.workflow_history[workflow_id] = review_history
```
Comment (vangheem) [inline castle/cms/_scripts/importjson.py]: this is safe?

Comment (tkimnguyen) [reply castle/cms/_scripts/importjson.py]: getChainFor() returns the workflow ID as a string. Normally we would not have more than 1 chain. It seems to work fine for the normal case and fits with the code that was there before that was taking the first element of the list returned by getChainFor().

Comment (vangheem) [reply castle/cms/_scripts/importjson.py]: I guess we'll never have more than one workflow assigned. If there are 2 assigned however, it seems to me like it'll set workflow history for each

Comment (tkimnguyen) [reply castle/cms/_scripts/importjson.py]: isn't that ok though? the import should result in content with matching metadata

Comment (tkimnguyen) [reply castle/cms/_scripts/importjson.py]: I remember Elro's presentation in DC about some financial instruments he created that had multiple workflows attached

Comment (tkimnguyen) [reply castle/cms/_scripts/importjson.py]: I've thought about it for OIE...

**Pull request merged**